src/main-process/parse-command-line.js

```diff
@@ -177,6 +193,22 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
+function readProjectSpecificationSync (filepath, executedFrom) {
+  try {
+    const readPath = path.isAbsolute(filepath) ? filepath : path.join(executedFrom, filepath)
```

Comment:
Should this conversion be moved to the caller?

Reply:
hey I asked for this too @philipfweiss!